edx_reverification_block/xblock/reverification_block.py

```diff
@@ -65,6 +68,13 @@ class ReverificationBlock(XBlock):
         help="ISO-8601 formatted string representing the due date of this related assessment."
     )
 
+    grace_period = Integer(
+        display_name="Verification Grace Period",
+        scope=Scope.settings,
+        default=ICRV_GRACE_PERIOD,
+        help="The number of days learners can verify their identity after the exam's due date."
```

Comment:
Just follow the existing pattern.  Where all fields are without transalation.
https://github.com/edx/edx-reverification-block/blob/master/edx_reverification_block/xblock/reverification_block.py#L30